frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -143,3 +143,16 @@ export const waitForStringNotInElement = (elem: ElementFinder, needle: string) =
     return !content.includes(needle);
   };
 };
+
+/**
+ * Search YAML manifest for a given string. Return true if found.
+ * @param     {string}    needle    String to search in YAML.
+ * @param     {string}    name      Name of the resource.
+ * @param     {string}    namespace Namespace of the resource.
+ * @param     {string}    kind      Kind of the resource.
+ * @returns   {boolean}             True if found, false otherwise.
+ */
+export function searchYAML(needle: string, name: string, namespace: string, kind: string): boolean {
+  const result = execSync(`oc get -o yaml -n ${namespace} ${kind} ${name}`).toString();
```

Comment:
I understand the power of a function like that but I am a little bit worried about this dependency on the `oc` tool.
It requires extra configuration step which should be documented.

At least path to it should be configurable (env variable?).

Anyway, I think it would be better to rely on existing API connection, i.e. via `k8sGet()` function. Wdyt?

Reply:
I'm not sure if it's necessary to use API because openshift core integration-tests use `kubectl` quite often. But I agree we should use `kubectl` rather than `oc` here.

Reply:
ok, makes sense. As long as we don't need specifics of the `oc`, the `kubectl` is better option. I see it used in the core tests now, so we would not introduce new dependency this way.

Reply:
> I am a little bit worried about this dependency on the `oc` tool.

> openshift core integration-tests use `kubectl` quite often.

@mareklibra @rhrazdil @jelkosz @spadgett 

Console integration tests should use `kubectl` instead of `oc` - this reflects the expectation that Console should work without issues on vanilla k8s clusters. (Console's primary target is OpenShift, but we want to keep compatibility with vanilla k8s as well.)

AFAIK, when you install `oc` it also creates a link to `kubectl` so the same tests should work for both vanilla k8s and OpenShift.

> As long as we don't need specifics of the `oc`, the `kubectl` is better option.

:+1: